packages/x-date-pickers/src/internals/components/PickersArrowSwitcher.tsx

```diff
@@ -7,9 +7,25 @@ import IconButton from '@mui/material/IconButton';
 import { ArrowLeft, ArrowRight } from './icons';
 
 export interface PickersArrowSwitcherSlotsComponent {
+  /**
+   * Button allowing to switch to the left view.
```

Comment:
The punctuation seems a bit weird.
In some descriptions we are putting `.` at the end, while in others not.
Could we settle on a one approach?